### ISSUE TITLE
Prompt “Test result not found.  ...” when click 'run', howerer it can work crorectly when click 'debug'

### ISSUE THREAD

**Issue by WakerCN** [vitest-dev/vscode#167]
同一个case
点击`run`按钮运行会报错，
点击`debug`按钮进行case测试可以正常的运行

操作gif如下图：
![vitest](https://github.com/vitest-dev/vscode/assets/50485199/3d89ec06-e247-4bfc-8571-a456b0549590)

下面是详细的报错信息：
![image](https://github.com/vitest-dev/vscode/assets/50485199/6909f16a-0809-480d-b4ae-ca1e3b01c858)

**Comment by IGx89** [comment]
Looks like you might be on Windows? If so you're likely running into the issue described in #160

**Comment by mbergal** [comment]
On Mac async test causes the same error on  "Run", works fine on "Debug":

```
import { test } from "vitest";

function delayPromise(ms: number) {
  return new Promise((resolve) => setTimeout(resolve, ms));
}

test("Captures", async () => {
  await delayPromise(1000);
}, 5000);
```

**Comment by likeuwill** [comment]
I had the same issue with monorepo
vitest.workspace
```
import { defineWorkspace } from 'vitest/config'

// defineWorkspace provides a nice type hinting DX
export default defineWorkspace([
  'apps/client/',
  {
    // add "extends" to merge two configs together
    extends: './vitest.config.ts',
    test: {
      include: ['tests/**/*.{browser}.spec.{ts,tsx,js}'],
      // it is recommended to define a name when using inline configs
      name: 'happy-dom',
      environment: 'happy-dom',
    }
  },
  {
    test: {
      include: ['tests/**/*.{browser}.spec.{ts,tsx,js}'],
      name: 'node',
      environment: 'node',
    }
  }
])
```
test
```
import { render } from '@testing-library/react'
import { describe, expect, it } from 'vitest'

import Logo from '../logo'

const renderComponent = () => render(<Logo />)

describe('Logo', () => {
  it('matches the snapshot', () => {
    const result = renderComponent()

    expect(result).toMatchSnapshot()
  })
})
```
vitest.config.ts in root
```
export default defineConfig({
  test: {
    globals: true,
    coverage: {
      provider: 'v8',
      reporter: ['lcovonly']
    }
  },
```

**Comment by sheremet-va** [comment]
This should be fixed in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.
